packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -20,6 +21,11 @@ export interface IDatePickerProps extends React.Props<DatePicker> {
    */
   calendarProps?: ICalendarProps;
 
+  /**
+  * Pass callout props to callout component
+  */
+  calloutProps?: ICallout;
```

Comment:
See my other comment, you are leaking the fact that the picker uses ICallout... If you want to set just a few props, may be best to just expose the props that you need to allow in date picker.